symfony/plugins/orangehrmLeavePlugin/config/routes.yaml

```diff
@@ -90,6 +90,15 @@ apiv2_leave_leave_entitlements:
   defaults:
     _api: OrangeHRM\Leave\Api\LeaveEntitlementAPI
 
+apiv2_leave_leave_balance:
+  path: /api/v2/leave/leave-balance/leave-type/{leaveTypeId}
```

Comment:
is that leave balance for logged-in user? what about 
`: /api/v2/leave/my-leave-balance/leave-type/{leaveTypeId`

Reply:
yes we can provide `empNumber` query parameter, depend on the permission return the leave balance

Reply:
ok fine